### ISSUE TITLE
Move exttp back into Modelica code

### ISSUE THREAD

**Issue by ibell** [ibell/coolprop#49]
@jowr What do you think about moving the EXTTP code just back into the Modelica code?  Originally that's what I thought we were going to do.  I suppose it is fine as is, but it also seems the enable/disable functions are not implemented in the CPSSI class, which gave me some compiler errors which is why they are currently commented out.  We need to figure out what to do here.

**Comment by jowr** [comment]
Yes, can do. I'll have a look after my holidays. @squoilin might also have a look at this if time allows.

**Comment by jowr** [comment]
I tried to fix it, but I had no success. Just tested on Windows for the first time and it doesn't work, works on Linux though...
